### PR TITLE
fix: disallow giving pegs to pockybot

### PR DESCRIPTION
**Pull request by Evangelinexx**
Seems to be an easy fix

### CODE REVIEW

**Review by Chromadream — APPROVED** [review]

File: src/PockyBot.NET/Services/Pegs/PegRequestValidator.cs [hunk 1]
```diff
@@ -54,6 +54,9 @@ private void ValidateRecipient(Message message)
                 throw new PegValidationException(
                     $"You can't peg yourself.");
             }
+            if (message.MessageParts[2].UserId == _settings.BotId){
```
Comment (jimcullenaus) [inline src/PockyBot.NET/Services/Pegs/PegRequestValidator.cs:57]: I'm astonished we hadn't done this already tbh

**Review by jimcullenaus — APPROVED** [review]

File: src/PockyBot.NET/Services/Pegs/PegRequestValidator.cs [hunk 2]
```diff
@@ -54,6 +54,9 @@ private void ValidateRecipient(Message message)
...
+                throw new PegValidationException("You can't give pegs to PockyBot.");
```
Comment (Lauraducky) [inline src/PockyBot.NET/Services/Pegs/PegRequestValidator.cs:58]: Is this gonna be returned to the user? Do we want to word it more like "You can't give pegs to me?"

Comment (Evangelinexx) [reply src/PockyBot.NET/Services/Pegs/PegRequestValidator.cs:58]: Hmm good idea

**Review by sholoms — DISMISSED** [review]

**Review by sholoms — APPROVED** [review]

**Review by rebecca-sell — DISMISSED** [review]

**Review by Lauraducky — APPROVED** [review]

**Review by sholoms — APPROVED** [review]

**Pull request merged**